src/ammo_effect.h

```diff
@@ -6,11 +6,11 @@
 #include <string>
 #include <vector>
 
+#include "field_type.h"
```

Comment:
It seems odd that you're adding this include when the only other change to this file is a deletion.

Reply:
This appears to be an attempt to address the current clang-tidy error about the thing that was deleted.

Reply:
Right, makes sense now.